src/main/java/org/openstreetmap/atlas/geography/atlas/pbf/store/PbfOneWay.java

```diff
@@ -80,7 +80,7 @@ else if (OneWayTag.isOneWayReversible(taggable))
         }
     }
 
-    private static boolean isNotAccessibleToVehicles(final Taggable taggable)
+    public static boolean isNotAccessibleToVehicles(final Taggable taggable)
```

Comment:
After the previous change to fix edge definition consistency (see #65), this method became unused. To avoid warnings, I made it public. I still think it serves a purpose for anyone looking for these tag combinations, but it may not be the place to put it. Candidate for removal.

Reply:
I would just remove it :)